### ISSUE TITLE
Build Error on Manjaro

### ISSUE THREAD

**Issue by aekst** [BoomingTech/Pilot#11]
报错信息为

```
.../Pilot/engine/bin/Linux/meta_parser: error while loading shared libraries: libclang.so.12: cannot open shared object file: No such file or directory
make[2]: *** [CMakeFiles/PilotPreCompile.dir/build.make:73: CMakeFiles/PilotPreCompile] Error 127
make[1]: *** [CMakeFiles/Makefile2:261: CMakeFiles/PilotPreCompile.dir/all] Error 2
```

在meta_parser同目录下可以看到libclang.so.12。可能meta_parser只是尝试寻找系统自带的libclang？当前manjaro的libclang版本为13

**Comment by WANGSSSSSSS** [comment]
this problem is due to your wrong ld library path configuration, thus you can add the following line in `build_linux.sh` file
`export export LD_LIBRARY_PATH= xxxxxx/Pilot/engine/bin/Linux` , replace xxxxx with your codebase path

**Comment by hyv1001** [comment]
This issue should be fixed by #53, please reopen it or create a new ticket if it still exists on your machine
